.env.production.sample

```diff
@@ -36,8 +36,8 @@ SMTP_LOGIN=
 SMTP_PASSWORD=
 SMTP_FROM_ADDRESS=notifications@example.com
 
-# Optional asset host for multi-server setups
-# CDN_HOST=assets.example.com
+# Optional asset host for multi-server setups (inc. protocol)
+# CDN_HOST=http://assets.example.com
```

Comment:
Very strange. I just checked, and this variable works without the protocol on mastodon.social.